float64 time
string cmd
float64 x
float64 y
float64 z
float64 vel

